Makefile

```diff
@@ -157,32 +157,27 @@ all: binaries
 android-dist: genconfig android
 
 # This is to be called within the docker image.
-docker-genassets: require-npm
+genassets: require-npm
```

Comment:
Good point, docker has a cost and it takes a lot of space. So, we could use docker only when building a distribution package, not for the daily development. That way we can be sure the final build is the same for Linux/Windows. How does it sound?

Perhaps it would be better to keep track of the solution for this on a different issue, it looks like the UI address issue does not depend on this problem.